web_pwa_oca/templates/assets.xml

```diff
@@ -2,23 +2,27 @@
 <odoo>
     <template id="web_layout_pwa" name="Web layout PWA" inherit_id="web.layout">
         <xpath expr="//meta[@name='viewport']" position="after">
-            <!-- Add link rel manifest -->
-            <link rel="manifest" t-attf-href="/web_pwa_oca/manifest.webmanifest"/>
-            <!-- Add iOS meta tags and icons -->
-            <meta name="apple-mobile-web-app-capable" content="yes"/>
-            <meta name="apple-mobile-web-app-status-bar-style" content="black"/>
-            <t t-set="pwa_name" t-value="request.env['ir.config_parameter'].sudo().get_param('pwa.manifest.name')"/>
-            <meta name="apple-mobile-web-app-title" t-att-content="pwa_name"/>
-            <link rel="apple-touch-icon" href="/web_pwa_oca/static/img/icons/icon-152x152.png"/>
-            <!-- Add meta theme-color -->
-            <t t-set="pwa_theme_color" t-value="request.env['ir.config_parameter'].sudo().get_param('pwa.manifest.theme_color')"/>
-            <meta name="theme-color" t-att-content="pwa_theme_color" />
+            <t t-if="not website">
```

Comment:
I think it would be better to change the "web.webclient_bootstrap" (t-set='head_web') template instead of "web.layout"...
what do you think?

Reply:
See comment below. I believe this should be moved to the top of web.assets_backend.
@pedrobaeza what do you think?

Reply:
My colleague is the expert on this, so I don't have a technical opinion on this. @Yajo can also give an opinion.